spec/tree/values/nodes/ParagraphSpec.js

```diff
@@ -0,0 +1,36 @@
+import TextContainer from "../../../../src/tree/values/nodes/TextContainer";
+import FormattingElement from "../../../../src/tree/values/FormattingElement";
+import Paragraph from "../../../../src/tree/values/nodes/Paragraph";
+
+describe( "Text tree node", () => {
+	it( "can make a new Paragraph tree node with paragraph opening tags", () => {
+		const phrasingElements = [
+			new FormattingElement( "a", 25, 29 ),
+		];
+		const text = "This is some text with a link.";
+		const textElement = new TextContainer( text, phrasingElements );
+
+		const openingTag = "<p>";
```

Comment:
The use of `<p>` again here does not seem optimal to me, since that is also the default.